tests/test_interface.py

```diff
@@ -42,6 +42,13 @@
 else:
   import unittest2 as unittest
 
+# Use external backport 'mock' on versions under 3.3
+if sys.version_info >= (3, 3):
+  import unittest.mock as mock
+
+else:
+  import mock
+
```

Comment:
(This question in no way interferes with this PR. As a point of general curiosity:
Is it better practice to test Python version numbers this way when determining what to import, or to try the first import and fail over to the second one? What if, say, Python 4.0 had been released before mock was added to unittest in Python 3.3? Should one just see if the duck quacks first?)

Reply:
IMHO both are fine. In this case I lean towards the version checking approach, because according to Python docs, mock is "*available as unittest.mock in Python 3.3 onwards"*. So if it fails with e.g. 4.0, it would be a Python (docs) bug.
It is important though that the unittests take all supported Python versions into consideration.

There might be a minimal performance difference between exception handling and tuple comparison, but I think we can disregard that.